include/swift/AST/ASTContext.h

```diff
@@ -489,6 +493,10 @@ class ASTContext final {
   /// Set a new stats reporter.
   void setStatsReporter(UnifiedStatsReporter *stats);
 
+  /// Set the module import callback function.
+  void setPreModuleImportCallback(
+      std::function<bool(llvm::StringRef, bool)> Callback);
```

Comment:
After having seen the LLDB patch — why not make this a `= {}` argument in the constructor?

Reply:
In my understanding, `std::function` creates an empty function by default, so no need to initialize it.

Reply:
I think Adrian is suggesting to make the Callback an optional argument in the constructor.